specification/network/resource-manager/Microsoft.Network/stable/2025-01-01/applicationGateway.json

```diff
@@ -3719,15 +3719,28 @@
           "type": "string",
           "description": "The string representation of the web application firewall rule sensitivity.",
           "enum": [
-            "None",
             "Low",
```

Comment:
Why is "None" being removed ? Will this not cause issues to existing customers who used this enum ?